mne/cov.py

```diff
@@ -1431,11 +1435,11 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings_cov_rank']
```

Comment:
Yeah, that would be a simple way to do it. Something like:
```
def _get_default(key):
   ...
```
@adykstra if you're sick of making changes we can merge this and I can do it.